certified-connectors/Pipeliner CRM/Readme.md

```diff
@@ -0,0 +1,51 @@
+# Pipeliner CRM
+Pipeliner is a next generation sales CRM software designed to empower sales teams and grow profitable customer relationships. The conector provides an API to work with Pipeliner records.
+
```

Comment:
Hello @vaclavcmar 

Please refer this readme.md file template link and update your readme.md file duly. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md